src/test/java/uk/ac/ebi/eva/pipeline/parameters/validation/InputStudyTypeValidatorTest.java

```diff
@@ -0,0 +1,117 @@
+import org.springframework.batch.core.JobParametersBuilder;
+import org.springframework.batch.core.JobParametersInvalidException;
+
+import uk.ac.ebi.eva.pipeline.parameters.JobParametersNames;
+
+public class InputStudyTypeValidatorTest {
+
+    private InputStudyTypeValidator validator;
+
+    @Before
+    public void setUp() throws Exception {
+        validator = new InputStudyTypeValidator();
+    }
+
+    @Test(expected = JobParametersInvalidException.class)
+    public void invalidStudyTypeShouldThrow() throws Exception {
+        JobParametersBuilder jobParametersBuilder = new JobParametersBuilder();
+        jobParametersBuilder.addString(JobParametersNames.INPUT_STUDY_TYPE, "invalid");
+        validator.validate(jobParametersBuilder.toJobParameters());
+    }
+
+    @Test(expected = JobParametersInvalidException.class)
+    public void emptyStudyTypeShouldThrow() throws Exception {
+        JobParametersBuilder jobParametersBuilder = new JobParametersBuilder();
+        jobParametersBuilder.addString(JobParametersNames.INPUT_STUDY_TYPE, "");
+        validator.validate(jobParametersBuilder.toJobParameters());
+    }
+
+    @Test
+    public void collectionStudyTypeIsValid() throws Exception {
+        JobParametersBuilder jobParametersBuilder = new JobParametersBuilder();
+        jobParametersBuilder.addString(JobParametersNames.INPUT_STUDY_TYPE, "COLLECTION");
+        validator.validate(jobParametersBuilder.toJobParameters());
+    }
+
+    @Test
+    public void familyStudyTypeIsValid() throws Exception {
+        JobParametersBuilder jobParametersBuilder = new JobParametersBuilder();
+        jobParametersBuilder.addString(JobParametersNames.INPUT_STUDY_TYPE, "FAMILY");
+        validator.validate(jobParametersBuilder.toJobParameters());
+    }
+
+    @Test
+    public void trioStudyTypeIsValid() throws Exception {
+        JobParametersBuilder jobParametersBuilder = new JobParametersBuilder();
+        jobParametersBuilder.addString(JobParametersNames.INPUT_STUDY_TYPE, "TRIO");
+        validator.validate(jobParametersBuilder.toJobParameters());
+    }
+
+    @Test
+    public void controlStudyTypeIsValid() throws Exception {
+        JobParametersBuilder jobParametersBuilder = new JobParametersBuilder();
+        jobParametersBuilder.addString(JobParametersNames.INPUT_STUDY_TYPE, "CONTROL");
+        validator.validate(jobParametersBuilder.toJobParameters());
+    }
+
+    @Test
+    public void caseStudyTypeIsValid() throws Exception {
+        JobParametersBuilder jobParametersBuilder = new JobParametersBuilder();
+        jobParametersBuilder.addString(JobParametersNames.INPUT_STUDY_TYPE, "CASE");
+        validator.validate(jobParametersBuilder.toJobParameters());
+    }
+
+    @Test
+    public void caseControlStudyTypeIsValid() throws Exception {
+        JobParametersBuilder jobParametersBuilder = new JobParametersBuilder();
+        jobParametersBuilder.addString(JobParametersNames.INPUT_STUDY_TYPE, "CASE_CONTROL");
```

Comment:
Nothing to change here, but I was just wondering if we should support more user-friendly strings like "case-control" and any possible capitalization of it.